ios/Podfile

```diff
@@ -63,6 +63,21 @@ abstract_target 'Status' do
   end
 
   post_install do |installer|
+    # https://github.com/facebook/react-native/blob/main/packages/react-native/scripts/react_native_pods.rb#L197-L202
+      react_native_post_install(
+        installer,
+        config[:reactNativePath],
+        :mac_catalyst_enabled => false,
+        # :ccache_enabled => true
+      )
```

Comment:
Wrong indent. Also no comment explaining why this is done. Just a RELATIVE link that targets `main` branch, which will become useless soon.